Source/carthage/Outdated.swift

```diff
@@ -112,3 +124,26 @@ public struct OutdatedCommand: CommandProtocol {
 			.waitOnCommand()
 	}
 }
+
+private func shouldIgnoreFetchingDependencies(fetchInterval: TimeInterval) -> SignalProducer<Bool, CarthageError> {
+    let key = "org.carthage.Constants.outdatedDependenciesLastFetchDate"
+
+    // When the fetch interval is not provided, clear the storage and fetch the dependencies.
+    guard fetchInterval > 0 else {
+        UserDefaults.standard.removeObject(forKey: key)
+        return SignalProducer<Bool, CarthageError>({ false })
+    }
+
+    // If the fetch interval time has not been set before, start measuring time from the current date.
+    guard let lastFetchDate = UserDefaults.standard.value(forKey: key) as? Date else {
+        UserDefaults.standard.set(Date(), forKey: key)
```

Comment:
As a storage, I decided to give a go for a `UserDefaults`. Is there any storage in Carthage code base which I should  have used? 🤔